notices/tests.py

```diff
@@ -1,3 +1,50 @@
 from django.test import TestCase
-
+from .models import Notice,NoticeBranchYear,BookmarkedNotice
+from profiles.models import FacultyDetail
+from django.contrib.auth.models import User
 # Create your tests here.
+
+#models test
+class NoticeTest(TestCase):
+    
+    def setUp(self):
+        user=User.objects.create(username='Ajay',password='5678')
+        faculty=FacultyDetail.objects.create(designation="teacher",department="IT",contact_no=9988213123,address="Indirapuram",alternate_email='abc@gmail.com',display_to_others='',user=user)
+        Notice.objects.create(faculty=faculty,title='extra-class',description='extra-class for compiler design will be from 1:30 to 2:30 tomorrow',file_attached='',subject='compiler design class',category='Academics')
+        
+    def test_create_notice(self):
+        p=User.objects.get(username='Ajay')
+        q=FacultyDetail.objects.get(user=p)
+        details=Notice.objects.get(faculty=q)
```

Comment:
please use meaningful variable names like `user`, `faculty_detail` and `notice` instead of `p`, `q` and `details`. Please correct this wherever you find this kind of mistake.